docs/architecture/decisions/0003-django.md

```diff
@@ -0,0 +1,35 @@
+# 3. Django
+
+Date: 2022-08-09
+
+## Status
+
+Accepted
+
+## Context
+
+The [Federal Audit Clearinghouse](https://github.com/GSA-TTS/FAC) project is a similar project to ours, from which we are taking many examples for reuse. FAC uses Django.
```

Comment:
I would maybe put this at the bottom. We were thinking of using Django independently but it is a happy coincidence that FAC uses it. Putting at the top might imply we are somehow tied to their technology choices. I think it is a plus but not the strongest reason.

Reply:
Good point. I've moved it.